firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nSuite1SnapshotTests.swift

```diff
@@ -146,14 +148,12 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
 
     @MainActor
     func testHistoryTableContextMenu() {
+        app.cells[AccessibilityIdentifiers.FirefoxHomepage.SearchBar.itemCell].waitAndTap()
         navigator.openURL(loremIpsumURL)
         waitUntilPageLoad()
         mozWaitForElementToNotExist(app.staticTexts["XCUITests-Runner pasted from Fennec"], timeout: 5)
         mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.settingsMenuButton], timeout: 10)
         waitForTabsButton()
-        navigator.performAction(Action.OpenNewTabFromTabTray)
```

Comment:
We do not need to go through a new tab to open the history page. Opening a new tab introduces some intermittent test failures. These steps do not affect the screenshot to be taken.